build.gradle.kts

```diff
@@ -43,4 +43,13 @@ allprojects {
          url = uri("https://oss.sonatype.org/content/repositories/snapshots")
       }
    }
+
+   tasks.withType<org.jetbrains.kotlin.gradle.tasks.KotlinCompile>().configureEach {
+      kotlinOptions {
+         freeCompilerArgs = freeCompilerArgs + "-Xopt-in=kotlin.RequiresOptIn"
+         jvmTarget = "1.8"
+         apiVersion = "1.6"
+         languageVersion = "1.6"
+      }
+   }
```

Comment:
Will Be get the following error, when it exec `./gradlew check`.
```
Error processing C:\Users\username\.gradle\caches\modules-2\files-2.1\org.bouncycastle\bcprov-jdk15on\1.68\46a080368d38b428d237a59458f9bc915222894d\bcprov-jdk15on-1.68.jar:META-INF\versions\15\org\bouncycastle\jcajce\provider\asymmetric\edec\BC11XDHPrivateKey.class: broken class file? (java.lang.IllegalArgumentException: Unsupported class file major version 59)
```

So I added the following.
https://github.com/kotest/kotest/blob/59726a2cbb44b890e207dcef2033811ae5029ea1/build.gradle.kts#L39-L46